src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.Managed.DesignTime.targets

```diff
@@ -396,6 +396,12 @@
           DependsOnTargets="GetSuggestedWorkloads"
           Condition="'@(MissingWorkloadPack)' != '' And '$(UsingMicrosoftNETSdk)' == 'true'" />
 
+  <!-- This target is used to collect the MissingSdkRuntime items in the project.-->
+  <Target Name="CollectMissingSdkRuntimes"
+          Returns="@(MissingSdkRuntime)"
+          DependsOnTargets="GetMissingSdkRuntimes"
+          Condition="'$(UsingMicrosoftNETSdk)' == 'true' and $([MSBuild]::VersionGreaterThanOrEquals('$(TargetFrameworkVersion)', '6.0'))" />
```

Comment:
>'6.0' [](http://example.com/codeflow?start=130&length=5)

This target will not be in .NET 6, will it? I assumed that .NET 7 would be the first version that could have such a target.

Reply:
Yes...for testing purposes it worked with .NET 6.
This code was deleted